cli/dts/lib.deno.ns.d.ts

```diff
@@ -108,6 +108,9 @@ declare namespace Deno {
      * after the test has exactly the same contents as before the test. Defaults
      * to true. */
     sanitizeResources?: boolean;
+
+    /** Ensure the test case does not accidentally prematurely exit. Defaults to true. */
+    sanitizeExit?: boolean;
```

Comment:
Do we need this option? I can't think of the case when this is set to false.

Reply:
Probably don't _really_ need it but I'd like to leave an escape hatch for the sake of completeness and just in case there's some scenario we haven't thought about.

Also serves as documentation, that this is sanitizer we have enabled by default.

Reply:
@caspervonb it'd be great to explicitly spell out in the docstring that it affects calls to `Deno.exit`

Reply:
Because this is changing the public API in the Deno namespace, I think we should be conservative here and leave it out. What about adding a TODO instead? If we need it we can come back to this PR and pull the diff.

Reply:
What about:

```js
/** Asserts that the test case does not call `Deno.exit`. Defaults to true. */
```